src/components/detail/ImageGallery.jsx

```diff
@@ -108,7 +113,8 @@ class ImageGallery extends Component {
 
         mediaCsids.forEach((mediaCsid, index) => {
           const altText = mediaAltTexts.get(index)
-            || intl.formatMessage(messages.defaultAltText, { num: index + 1 });
+            || title ? intl.formatMessage(messages.titledAltText, { title, num: index + 1 })
+            : intl.formatMessage(messages.defaultAltText, { num: index + 1 });
```

Comment:
This doesn't do what you want, because `||` has higher precedence than `?`. You could put parens around the ternary expression, but I prefer not to combine boolean and ternary (or to nest ternaries), because it's hard to read.  I think the best is something like
```
let altText = mediaAltTexts.get(index);

if (!altText) {
  altText = ...
}
```

Reply:
Ah, I originally had it with parens when testing but didn't realize they were necessary and had removed them. I'll update this shortly.